src/Chapter19/Cryptographer.cs

```diff
@@ -27,9 +27,9 @@ public Cryptographer()
         }
         #endregion CONSTRUCTORS
 
-        public string Encrypt(string text)
+        public byte[] Encrypt(string text)
         {
-            return Encoding.UTF8.GetString(EncryptAsync(text).Result);
+            return EncryptAsync(text).Result;
```

Comment:
Is this code shown in the book? This can cause a deadlock if this code is invoked from a program with a synchronization context. If we have to force sync over async, this is a better pattern as it forces the execution onto a background thread.

```suggestion
            return Task.Run(() => EncryptAsync(text)).Result;
```

Reply:
It's a console program so there is no synchronization context, however, you make a good point.  I will document this as something to review when I get to that chapter.